Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterCrashReport.kt

```diff
@@ -0,0 +1,26 @@
+@file:JvmName("FormatterCrashReport")
+
+package de.rki.coronawarnapp.util.formatter
+
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import de.rki.coronawarnapp.R
+import org.joda.time.DateTimeZone
+import org.joda.time.Instant
+
+@BindingAdapter("crashReportTitle")
+fun formatCrashReportId(textView: TextView, id: Int) {
```

Comment:
Create a State object representing the UI, and let that implement the necessary formatting code.

We want to move away from `FormatterHelpers` :wink:

Reply:
Or if it's simple enough, do the text formatting directly in the ViewHolder/Fragment

Reply:
all right, will do.

Reply:
adjusted.